### PR TITLE
Add a hover effect to nav-links in the nav-bar

### LINKED ISSUE #36: Add a hover effect to the links in the navbar
**Is your feature request related to a problem? Please describe.**
Currently there is no hover effect in the links of the navbar

**Describe the solution you'd like**
Add a hover effect to the links in the navbar

### PR DESCRIPTION
**Pull request by YoshithaRathnayake**
## Purpose

The purpose of this PR is to fix #36 

## Goals

Adding a hover effect to nav-links in the nav-bar

## Approach

I added a hover effect to nav-links in the nav-bar

## Preview Link
https://yoshitharathnayake.github.io/sef-dev/

### Screenshots

image

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: custom-styles.css [hunk 1]
```diff
@@ -1,3 +1,13 @@
...
+
+.nav-link-hover {
+    padding: 15px;
+}
+
+.nav-link-hover:hover {
+    background-color: rgb(71, 71, 71);
+    padding: 15px;
+    border-radius: 5px;
+}
```
Comment (anjula-sack) [inline custom-styles.css:13]: Add the end of line

**Review by anjula-sack — CHANGES_REQUESTED** [review]
Awesome! minor change is requested

**Comment by YoshithaRathnayake** [comment]
> Awesome! minor change is requested

sure

**Review by anjula-sack — APPROVED** [review]
Thank you! @YoshithaRathnayake

**Pull request merged**